salt/states/boto_secgroup.py

```diff
@@ -577,16 +577,19 @@ def absent(
     '''
     ret = {'name': name, 'result': True, 'comment': '', 'changes': {}}
 
-    sg = __salt__['boto_secgroup.get_config'](name, True, region, key, keyid,
-                                              profile, vpc_id, vpc_name)
+    sg = __salt__['boto_secgroup.get_config'](name=name, group_id=None, region=region, key=key,
```

Comment:
I may be mistaken, but I was thinking `True` being there was an error. I made it consistent with the other calls to boto_secgroup.get_config where group_id defaults to `None`

`boto_secgroup.py:296`
```
def get_config(name=None, group_id=None, region=None, key=None, keyid=None,
               profile=None, vpc_id=None, vpc_name=None):
```

If that is not the case, what would passing `True` accomplish here? Just for my understanding.